tea-config-generator/run.sh

```diff
@@ -0,0 +1,182 @@
+  echo '  ./run.sh -[o|I]'
+  echo
+
+  format="%4s %-8s %10s | %s\n"
+  printf "${format}" 'flag' 'value' 'name' 'Description'
+  printf "${format}" '----' '--------' '------' '-------------------------'
+  printf "${format}" '-h' '' 'Help' 'Print out a help document'
+  printf "${format}" '-c' '' 'Color on' 'Turn color on (default)'
+  printf "${format}" '-C' '' 'Color off' 'Turn color off'
+  printf "${format}" '-d' '' 'Documentation' 'Generate Documentation for AWS'
+  printf "${format}" '-D' '' 'Deploy' 'Run deployment tasks'
+  printf "${format}" '-u' '' 'Unittest' 'Run python unit test'
+  printf "${format}" '-U' '' 'Unittest' 'Run python unit test and save results'
+  printf "${format}" '-j' '' 'JUnittest' 'Run python unit test and save junit results'
+  printf "${format}" '-l' '' 'Lint' 'Run pylint over all files'
+  printf "${format}" '-L' '' 'Lint' 'Run pylint over all files and save results'
+  printf "${format}" '-t' '<token>' 'Token' 'Set token'
+  printf "${format}" '-r' '' 'Report' 'Generate code coverage report'
+  printf "${format}" '-e' '' 'Example' 'Do curl examples'
+  printf "${format}" '-o' '' 'Offline' 'Run serverless offline then exit'
+  printf "${format}" '-I' '' 'Install' 'Install dependent libraries'
+}
+
+while getopts 'hcCuUlLjt:oreIxD' opt; do
+  case ${opt} in
+    h) help_doc ;;
+    c) color_mode='yes';;
+    C) color_mode='no' ;;
+    d) documentation ;;
+    D) deploy ;;
+    u) python3 -m unittest discover -s ./ -p '*test.py' ;;
+    U) python3 -m unittest discover -s ./ -p '*test.py' &> test.results.txt ;;
+    l) lint ;;
+    L) lint &> list.results ;;
+    j) pip3 install pytest ; py.test --junitxml junit.xml ;;
+    t) token=${OPTARG} ;;
+    o) serverless offline ; exit ;;
+    r) report_code_coverage ;;
+    
+    x) rm -rf build script.log ;;
+
+    e)
+      baseEndPoint='http://localhost:3000/dev'
+      cprintf $YELLOW 'Example calls'
+      
+      echo
+      cprintf $GREEN "Calling /tea"
+      curl -s -H 'Cmr-Pretty: true' "${baseEndPoint}/configuration/tea"
+      echo
+      cprintf $GREEN '----'
+      read
+      
+      echo
+      cprintf $GREEN 'Calling /tea/capabilities'
+      curl -s "${baseEndPoint}/configuration/tea/capabilities" | head
+      echo
+      cprintf $GREEN '----'
+      read
+      
+      echo
+      cprintf $GREEN 'Calling /tea/status'
+      curl -i "${baseEndPoint}/configuration/tea/status"
+      echo
+      cprintf $GREEN '----'
+      
+      echo
+      cprintf $GREEN 'Calling /tea/provider/POCLOUD, may take 7 seconds'
+      curl -H "Cmr-Token: ${token}" "${baseEndPoint}/configuration/tea/provider/POCLOUD"
+      echo
+      cprintf $GREEN '----'
+
+      echo
+      cprintf $GREEN 'Calling /tea/provider/POCLOUD again with just headers, may take 7 seconds'
+      curl -I -H "Cmr-Token: ${token}" "${baseEndPoint}/configuration/tea/provider/POCLOUD"
+      echo
+
+      #curl -s http://localhost:7000/local-bucket/index.html
```

Comment:
Comments should be removed here

Reply:
I removed most, left a comment for the ones that I want in the file for documentation